.eslintrc

```diff
@@ -0,0 +1,39 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "plugins": [
+    "import",
+    "jsx-a11y",
+    "react",
+    "react-hooks"
+  ],
+  "env": {
+    "browser": true,
+    "jest": true
+  },
+  "rules": {
+    "arrow-parens": ["error", "as-needed"],
+    "comma-dangle": ["error", "always-multiline"],
+    "import/no-cycle": "error",
+    "import/prefer-default-export": "off",
```

Comment:
Du coup plutôt `export default` ? J'avais eu le débat avec un collègue et on était plutôt parti sur des exports nommés, mais la [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) dit :

> Named exports are useful to export several values. During the import, one will be able to use the same name to refer to the corresponding value.
> 
> Concerning the default export, there is only a single default export per module. A default export can be a function, a class, an object or anything else. This value is to be considered as the “main” exported value since it will be the simplest to import.

Du coup, le `default` pour les composants me semble bien et le named pour les utils... ?

Reply:
Je suis aussi plutôt pour les exports nommés, notamment parce que tu es obligé de garder le même nom à l'import (ça évite les imports avec des fautes de frappe aléatoires).

Du coup, la doc me perturbe :thinking: